thrift0.13/src/main/java/com/linecorp/armeria/common/thrift/ThriftSerializationFormats.java

```diff
@@ -15,16 +15,10 @@
  */
 package com.linecorp.armeria.common.thrift;
 
-import static java.util.Objects.requireNonNull;
-
-import java.util.Set;
-
-import com.google.common.collect.ImmutableSet;
-
 import com.linecorp.armeria.common.SerializationFormat;
 
 /**
- * Thrift-related {@link SerializationFormat} instances.
+ * Out-of-the box supported Thrift-related {@link SerializationFormat} instances.
```

Comment:
How about just modifying this class class to make `values()` return all discovered Thrift serialization formats?

Reply:
Understood. It makes more sense.